playbooks/openstack/resources.py

```diff
@@ -182,9 +190,8 @@ def build_inventory():
 
 def _get_stack_outputs(cloud_client):
     """Returns a dictionary with the stack outputs"""
-    cluster_name = os.getenv('OPENSHIFT_CLUSTER', 'openshift-cluster')
 
-    stack = cloud_client.get_stack(cluster_name)
+    stack = cloud_client.get_stack(OPENSHIFT_STACK_NAME)
```

Comment:
Please revert this back to `OPENSHIFT_CLUSTER` here. The stack name is not configurable separately and OPENSHIFT_CLUSTER should always match the stack name.